pandas/core/window.py

```diff
@@ -0,0 +1,1361 @@
+
+    def __init__(self, obj, com=None, span=None, halflife=None, min_periods=0, freq=None,
+                 adjust=True, ignore_na=False, axis=0):
+        self.obj = obj
+        self.com = _get_center_of_mass(com, span, halflife)
+        self.min_periods = min_periods
+        self.freq = freq
+        self.adjust = adjust
+        self.ignore_na = ignore_na
+        self.axis = axis
+
+    @property
+    def _constructor(self):
+        return EWM
+
+    @Substitution(name='ewm')
+    @Appender(SelectionMixin._agg_doc)
+    @Appender(SelectionMixin._see_also_template)
+    def aggregate(self, arg, *args, **kwargs):
+        return super(EWM, self).aggregate(arg, *args, **kwargs)
+
+    agg = aggregate
+
+    def _apply(self, func, how=None, **kwargs):
+        """Rolling statistical measure using supplied function. Designed to be
+        used with passed-in Cython array-based functions.
+
+        Parameters
+        ----------
+        func : string/callable to apply
+        how : string, default to None (DEPRECATED)
+            how to resample
+
+        Returns
+        -------
+        y : type of input argument
+
+        """
+        blocks, obj = self._create_blocks(how=how)
+        results = []
+        for b in blocks:
+            try:
+                values = self._prep_values(b.values)
+            except TypeError:
+                results.append(b.values.copy())
+                continue
+
+            if values.size == 0:
+                results.append(values.copy())
+                continue
+
+            # if we have a string function name, wrap it
+            if isinstance(func, compat.string_types):
+                if not hasattr(algos, func):
+                    raise ValueError("we do not support this function algos.{0}".format(func))
+
+                cfunc = getattr(algos, func)
+                def func(arg):
+                    return cfunc(arg, self.com, int(self.adjust), int(self.ignore_na), int(self.min_periods))
+
+            results.append(np.apply_along_axis(func, self.axis, values))
+
+        return self._wrap_results(results, blocks, obj)
+
+    @Substitution(name='ewm')
+    @Appender(_doc_template)
+    def mean(self):
+        """exponential weighted moving average"""
+        return self._apply('ewma')
+
+    @Substitution(name='ewm')
+    @Appender(_doc_template)
+    def std(self, bias=False):
+        """exponential weighted moving stddev
+
+        Parameters
+        ----------
+        bias : boolean, default False
+           Use a standard estimation bias correction
+        """
+        return _zsqrt(self.var(bias=bias))
+    vol=std
+
+    @Substitution(name='ewm')
+    @Appender(_doc_template)
+    def var(self, bias=False):
+        """exponential weighted moving average
+
+        Parameters
+        ----------
+        bias : boolean, default False
+           Use a standard estimation bias correction
+        """
```

Comment:
There is something strange with this docstring:

```
`In [85]: pd.core.window.EWM.var?
Signature: pd.core.window.EWM.var(self, bias=False)
Docstring:
exponential weighted moving average

        Parameters
        ----------
        bias : boolean, default False
           Use a standard estimation bias correction


Returns
-------
same type as input

See also
--------
`pandas.Series.ewm`
`pandas.DataFrame.ewm`
File:      c:\users\vdbosscj\scipy\pandas-joris\pandas\core\window.py
```

For some reason the Parameters are indented, although this is just a plain docstring? And the same approach with appending works in other places just fine